File: b_rank_new_level_up_problems/b_rank_new_level_up_problems__get_one.cs
// 実行時間 20ms
using static System.Console;
using static System.Linq.Enumerable;

internal class Program
{
    static void Main()
    {
        int[] hw = ReadLine().Split().Select( int.Parse ).ToArray();
        string[] s = new string[hw[0]];

        foreach( int i in Range( 0, hw[0] ) ) {
            s[i] = ReadLine();
        }

        int[] yx = ReadLine().Split().Select( int.Parse ).ToArray();
        char[] c = s[yx[0]].ToCharArray();

        c[yx[1]] = c[yx[1]] == '.' ? '#' : '.';
        s[yx[0]] = string.Join( "", c );
        WriteLine( string.Join( System.Environment.NewLine, s ) );
        return;
    }
}

File: b_rank_new_level_up_problems/b_rank_new_level_up_problems__get_one.py
# 実行時間 20ms
h, w = list(map(int, input().split()))
a = [[False for i in range(w)] for j in range(h)]

for i in range(h):
    s = input()

    for j, c in enumerate(s):
        if c == "#":
            a[i][j] = True

y, x = list(map(int, input().split()))

a[y][x] = not a[y][x]
for i in range(h):
    for j in range(w):
        print("#" if a[i][j] else ".", end="")
    print()
